arbitrum-docs/build-decentralized-apps/how-tos/migrate-nfts-arbitrum.mdx

```diff
@@ -0,0 +1,137 @@
+---
+id: migrate-nfts-arbitrum
+title: "How to migrate your NFT collection to Arbitrum"
+sidebar_label: "Migrate your NFT collection to Arbitrum"
+description: Learn how to migrate your NFT collection to Arbitrum through third-party bridges and DIY bridging solutions.
+author: amarrazza
+content-type: how-to
+reader_audience: Developers who want to migrate their NFT collections to Arbitrum from either Ethereum mainnet or other non-EVM chains.
+---
+
+
+import PublicPreviewBannerPartial from '../../partials/_public-preview-banner-partial.md'; 
+
+<PublicPreviewBannerPartial />
+
+This how-to is for **developers** who want to migrate existing <a data-quicklook-from="non-fungible-tokens-nfts">NFT</a> collections to <a data-quicklook-from="arbitrum-one">Arbitrum One</a> or <a data-quicklook-from="arbitrum-nova">Nova</a>.
+
+This guidance applies to NFT collections used within web3 games, decentralized apps (dApps), and other use-cases.
```

Comment:
When I read this, I have the impression that not all NFT collections can be migrated. There are many types of NFTs that are not included there and I think there's no limitation for them: PFP/brand NFTs, music NFTs, DeFi NFTs (these could be included in dApps, but the latter looks too general, imo), ...

Reply:
This makes sense - we can use the prerequisites section to articulate qualifiers if needed... will remove this :)